client/src/Dashboard/Dashboard.js

```diff
@@ -252,7 +274,15 @@ export function Dashboard() {
   return (
     <div className="dashboard sm:mx-8">
       <div className="dashboard-title m-2">
-        <Typography.Title>{t('dashboardTitle')}</Typography.Title>
+        <div className="flex items-center mb-3">
+          <Typography.Title className="dashboard-title mr-4">
+            {t('dashboardTitle')}
+          </Typography.Title>
+          <Button className="date-filter-button mr-2 text-base py-2 px-4">
```

Comment:
```suggestion
          <Button className="date-filter-button mr-2 text-base py-2 px-4" disabled>
```
@rebeccakarasiktw might want this button disabled until it actually does something, in the spirit of iterative development.  That means the styling will be gray though, so that might not be desired.  If she's okay with it, I'm 👍🏼

Reply:
makes sense to me to just go ahead and add the disabled attribute until it does something! will add.

Reply:
just checked this out -- adding the disabled attr adds a bunch of ant-design styling. the way I have it right now, the button is essentially disabled, bc there is no hover, active effect and no associated onClick event. it just looks like a static rectangle UI. If we want to add the 'disabled' attr, I am sure I can figure out how to override the styles.